modules/standard/BigInteger.chpl

```diff
@@ -1323,7 +1323,15 @@ module BigInteger {
   }
 
 
-  // Special Operations
+  /*  Returns the Jacobi symbol ``a/b``, which is definied only when ``b`` is odd.
+
+      Utilizes the GMP function `mpz_jacobi
+      <https://gmplib.org/manual/Number-Theoretic-Functions>`_.
+
+      :return: the Jacobi symbol
+      :rtype: ``int``
+  */
+  @unstable("jacobi is unstable and may move in the future")
```

Comment:
Is all that unstable means is that something may move in the future? I'm not sure we know for sure what might happen to any of these unstable methods, we just reserve the right to do whatever we want with them, so I think the wording "may move" sounds odd to me. Something like "may change" or "may be removed" or something makes more sense to me. What do you think?

Reply:
I agree and I think I prefer "may change" is probably best since we reserve the right to do whatever we want with them